diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/WhiteSpaceRule.kt

```diff
@@ -160,10 +164,18 @@ class WhiteSpaceRule(private val configRules: List<RulesConfig>) : Rule("horizon
         }
     }
 
+    private fun handleRbrace(node: ASTNode) {
+        if (node.treeParent.elementType == FUNCTION_LITERAL && !node.treePrev.isWhiteSpace()) {
+            WRONG_WHITESPACE.warnAndFix(configRules, emitWarn, isFixMode, "there should be a whitespace before }", node.startOffset, node) {
+                node.treeParent.addChild(PsiWhiteSpaceImpl(" "), node)
```

Comment:
Are you sure this white space normally belongs to `FUNTION_LITERAL` and not to `BLOCK`? I remember we had some issues with something similar in the past.

Reply:
I checked the tree of the correct code and it was like this.

Reply:
I mean the `WHITE_SPACE` was right after `FUNCTION_LITERAL`

Reply:
kk, good